### DISCUSSION TITLE
Advice needed - 10+ panels in new house - flash now or after install?

### DISCUSSION THREAD

**Discussion by henkedk** [Blackymas/NSPanel_HA_Blueprint#1972] [Q&A]
Hi guys,
I'm getting a new house built, my HA is not on site yet (nor is any network etc) but the installer ofc wants to put the panels to make sure the HW-button wires function.
What's my best option? Will the HW buttons work after a fresh flash (without TFT firmware?) or should I wait till after I move in (and be forced to dismantle installations to safely flash the devices?

**Comment by HKUser1** [comment]
I would buy for testing a cheap normal switch... you have to invest 5€ and the NS Panels stay safe for flashing....
